### DISCUSSION TITLE
add mild coaster scream and laugh to spiral slide

### DISCUSSION THREAD

**Discussion by 733737** [OpenRCT2/OpenRCT2#18130] [Feature requests]
Can we add more sounds to the spiral slide? Right now guests go down it silently, which I feel is unrealistic

**Comment by jjpb44** [comment]
Good point, I always wondered why this ride is so quiet, this could apply to other static rides as well. What happens is that those screams and laughs cannot occur in each visitor, the frequency has to be too low so as not to saturate the laughs and screams of other attractions (roller coasters)
